packages/blocks/styles/components/t-table.css

```diff
@@ -109,3 +109,50 @@
          absolute
          right-6;
 }
+
+.sortable__th__arrow-wrapper {
+  display: inline-block;
+  cursor: pointer;
+}
+
+.sortable__th__arrow {
+  margin-left: 5px;
+  border-left: 4px solid rgba(0, 0, 0, 0);
+  border-right: 4px solid rgba(0, 0, 0, 0);
+  width: 0px;
+  height: 0px;
+  font-size: 0px;
+  line-height: 0;
+  content: '';
+  border-top-style: solid;
+  border-bottom-style: solid;
+}
+
+.sortable__th__arrowup {
+  border-bottom-width: 4px;
+  border-bottom-color: #cccccc;
+  border-top: none;
+}
+
+.sortable__th__arrowdown {
+  margin-top: 3px;
+  border-top-width: 4px;
+  border-top-color: #cccccc;
+  border-bottom: none;
+}
+
+table th.up .sortable__th__arrowdown {
+  border-top-color: transparent;
+}
+
+table th.down .sortable__th__arrowup {
+  border-bottom-color: transparent;
+}
+
+table th.up .sortable__th__arrow {
+  border-bottom-color: #00a8ea;
+}
+
```

Comment:
Looks great 👍 

<img width="120" alt="image" src="https://user-images.githubusercontent.com/36261498/171166412-41826cfe-d48b-4e64-a887-aec8d8b8a1f3.png">

Maybe we can center align the arrow when sorted,  or alternatively keep the "other direction arrow" in a muted color like this, what do you think?

<img width="141" alt="image" src="https://user-images.githubusercontent.com/36261498/171167915-eebf9392-91ea-4d46-bb5d-862691a53eaa.png">

Reply:
Good idea!